rs/registry/canister/src/mutations/do_update_subnet.rs

```diff
@@ -250,27 +250,23 @@ impl Registry {
     }
 
     /// Validates that the SEV feature is not changed on an existing subnet.
+    ///
     /// Panics if the SEV feature is attempted to be changed.
```

Comment:
NCR, because it was already like this.

This seems like an arbitrary limitation. I would only do this if being able to change after the fact would **_actually_** break something. Maybe that is the case. Regardless, it would be nice if this explained why that is banned.